### ISSUE TITLE
Área para listar os eventos

### ISSUE THREAD

**Issue by luizcarvalho** [startup-tocantins/startup-tocantins.github.io#14]
Pensei em colocar no Medium também.

**Comment by vaniusg** [comment]
Eventos previstos para 2018:

SW Araguaina
SW Gurupi
SW Palmas
Muitos talks

**Comment by luizcarvalho** [comment]
@vaniusg era mais questão de qual área do site colocaríamos isso ou como colocaríamos isso. Não quais os eventos :P

**Comment by vaniusg** [comment]
Pensei q já era sugestões para o texto em si... como disse, usei muito pouco o github

**Comment by luizcarvalho** [comment]
hahahah não @vaniusg na verdade o problema não é o github, foi meu texto que não ficou claro! Culpa foi minha.

**Comment by luizcarvalho** [comment]
Em eventos, a princípio, pensei em colocar um embed do Google Agenda, com uma agenda específica para isso.

https://support.google.com/calendar/answer/41207?hl=pt-BR

Ai ficaria em uma página interna separada.

**Comment by vaniusg** [comment]
Fica muito mais fácil de gerenciar!!!

**Comment by luizcarvalho** [comment]
Boaaaaa, faremos isso então.

**Comment by gcsgpp** [comment]
Concordo com essa ideia do Google Agendas! 😄 

Sobre os SWs vi que tem o SW Palmas 2017 está apontando para a página do FB e as edições anteriores estão sem links. O que acham de apontar para as páginas da techstars de cada um? Esses links:

2017 - http://communities.techstars.com/brazil/palmas/startup-weekend/11294
2016 - http://communities.techstars.com/brazil/palmas/startup-weekend/9406
2015 - http://communities.techstars.com/brazil/palmas/startup-weekend/6843
2014 - http://communities.techstars.com/brazil/palmas/startup-weekend/4372

**Comment by luizcarvalho** [comment]
Concordo demais... e de inclusive adicionar os Weekends na página :P.

**Comment by gcsgpp** [comment]
@luizcarvalho já tem lá no www.startupto.com.br

**Comment by vaniusg** [comment]
Me passe o codigo que acrescento lá.

**Comment by gcsgpp** [comment]
@vaniusg , o email @startupto.com.br é Google? Pq se for, acho que seria interessante utilizar ele para criar o calendário. Se não, acho que poderia ser utilizado aquele startuptocantins@gmail.com (ou algo similar), o que acha @luizcarvalho ?

**Comment by luizcarvalho** [comment]
Não é Google não @gcsgpp , vamos criar com o @gmail mesmo! BTF!

**Comment by guilhermebferreira** [comment]
vou precisar de mais tempo pra subir a próxima melhoria na sessão de eventos, houve vários conflitos entre o css que o Arthur fez, com o pré existente do site, vou precisar ajeitar isso com calma.
